internal/server/server.go

```diff
@@ -30,9 +32,21 @@ func initialDatabase() {
 // Server method handles all requests
 func Server(port string) {
 	engine := html.New("internal/handlers/views", ".html")
+	var errMessage models.Error
 	
 	app := fiber.New(fiber.Config{
 		Views: engine,
+		ErrorHandler: func(ctx *fiber.Ctx, err error) error {
+			if errors.Is(err, sql.ErrNoRows)  {
```

Comment:
this line looks correct to me - does it not work?

Reply:
Yes it does.